365/client/html/splash.html

```diff
@@ -42,4 +106,22 @@ <h1 id="splashHead">365</h1>
 
 <template name="loggingIn">
   Logging in...
-</template>
\ No newline at end of file
+</template>
+
+<template name="splashBanner">
+	<div id="splashBannerHeader">
+		<img id="splashBannerLogo" src="images/keepspace-logo-blue.png">
+		<div id="centerDiv">
+			<input class="headerEmail" id="betaEmail" type="email" name="betaEmail"
+	        placeholder="ENTER E-MAIL">
+	        <input type="button" id="checkButton" value="✓" />
+	        <p id="betaLink" class="splashBannerP">SIGN UP FOR BETA</p>
+    	</div>
+    	<div id="linkDiv">
+	        <p id="aboutLink" class="splashBannerP">ABOUT</p>
+	        <p id="teamLink" class="splashBannerP">TEAM</p>
+	        <p id="loginLink" class="splashBannerP">LOG IN</p>
+    	</div>
+	</div>
+	{{> yield}}
+</template>
```

Comment:
Can you consolidate splash.html and login.html, they're both for the same page but we don't want the login page's templates scattered across multiple files.. keep whichever one you want